src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4encoder/HaivisionX4EncoderCommunicator.java

```diff
@@ -770,7 +1100,9 @@ private void extractPortNumberList(String portNumber) {
 						Integer.parseInt(portNumberItem.substring(index + 1));
 						portNumberRangeList.add(portNumberItem);
 					} catch (Exception ex) {
-						throw new ResourceNotReachableException("The port range not correct format" + portNumberItem);
+						if (logger.isDebugEnabled()) {
+							logger.debug("The port range not correct format" + portNumberItem);
```

Comment:
Please don't swallow exception...`logger.debug("The port range not correct format" + portNumberItem, ex)`

Reply:
It's the behavior that I'm expected because I want to ignore the input that has invalid format and keep the rest.